src/CssToInlineStyles.php

```diff
@@ -74,48 +78,78 @@ class CssToInlineStyles
      * Creates an instance, you could set the HTML and CSS here, or load it
      * later.
      *
-     * @return void
-     * @param  string [optional] $html The HTML to process.
-     * @param  string [optional] $css  The CSS to use.
+     * @param  null|string $html [optional] The HTML to process.
+     * @param  null|string $css  [optional] The CSS to use.
      */
     public function __construct($html = null, $css = null)
     {
         if ($html !== null) {
             $this->setHTML($html);
         }
+
         if ($css !== null) {
             $this->setCSS($css);
         }
     }
 
     /**
-     * Cleanup the generated HTML
+     * Set HTML to process
      *
-     * @return string
-     * @param  string $html The HTML to cleanup.
+     * @return void
+     *
+     * @param  string $html The HTML to process.
      */
-    private function cleanupHTML($html)
+    public function setHTML($html)
```

Comment:
changing the order of functions should not be done in the same PR than adding a new feature. It makes the review really hard because you are both reordering the existing code and changing some code